test/jit/test_cuda.py

```diff
@@ -49,6 +49,74 @@ def tearDown(self):
         torch.cuda.empty_cache()
         super(TestCUDA, self).tearDown()
 
+    def test_stream_args(self):
+        # Test stream creation with default arguments
+        @torch.jit.script
+        def stream_default_args() -> bool:
+            s = torch.jit.cuda.Stream()
+            return s.device_index() == torch.cuda._current_device()
+
+        @torch.jit.script
+        def stream_default_args_1() -> bool:
+            s = torch.jit.cuda.Stream(priority=0)
+            return s.device_index() == torch.cuda._current_device()
+
+        @torch.jit.script
+        def stream_default_args_2() -> bool:
+            d = torch.device("cuda:1")
+            s = torch.jit.cuda.Stream(d)
+            return s.device_index() == 1
+
+        @torch.jit.script
+        def stream_args() -> bool:
+            d = torch.device("cuda:0")
+            s = torch.jit.cuda.Stream(d, 0)
```

Comment:
Is there a way to test that the priority is being honoured?

Reply:
No I don't think so we check the priority